mhs/outbound/main.py

```diff
@@ -101,14 +101,25 @@ def start_tornado_server(data_dir: pathlib.Path, workflows: Dict[str, workflow.C
     # Note that the paths in generate_openapi.py should be updated if these
     # paths are changed
     supplier_application = tornado.web.Application(
-        [(r"/", client_request_handler.SynchronousHandler,
-          dict(config_manager=config_manager, workflows=workflows)),
-         (r"/healthcheck", healthcheck_handler.HealthcheckHandler)])
+        [
+            (r"/", client_request_handler.SynchronousHandler,dict(config_manager=config_manager, workflows=workflows)),
+            (r"/healthcheck", healthcheck_handler.HealthcheckHandler)
+        ])
     supplier_server = tornado.httpserver.HTTPServer(supplier_application)
-    supplier_server.listen(80)
+    server_port = int(config.get_config('OUTBOUND_SERVER_PORT', default='80'))
```

Comment:
No env var in cfg as the default port 80 is ok and doesn't break stuff in dockers